site/src/pages/TemplateSettingsPage/TemplateSettingsForm.tsx

```diff
@@ -20,16 +21,24 @@ export const Language = {
   descriptionLabel: "Description",
   maxTtlLabel: "Auto-stop limit",
   iconLabel: "Icon",
-  // This is the same from the CLI on https://github.com/coder/coder/blob/546157b63ef9204658acf58cb653aa9936b70c49/cli/templateedit.go#L59
-  maxTtlHelperText: "Edit the template maximum time before shutdown in seconds",
   formAriaLabel: "Template settings form",
   selectEmoji: "Select emoji",
+  ttlMaxError: "Please enter a limit that is less than or equal to 168 hours (7 days).",
+  descriptionMaxError: "Please enter a description that is less than or equal to 128 characters.",
+  ttlHelperText: (ttl: number): string =>
+    `Workspaces created from this template may not remain running longer than ${ttl} hours.`,
```

Comment:
same comment as above 👆🏼 use punctuation or don't (your call) but hoping we can promote consistency